lib/begin.go

```diff
@@ -188,6 +189,22 @@ func (a *ACBuild) writeEmptyManifest() error {
 		return err
 	}
 
+	archvalue := runtime.GOARCH
+	if runtime.GOOS == "linux" && archvalue == "arm" {
+		archvalue = "armv7l"
+		walker := func(path string, info os.FileInfo, err error) error {
```

Comment:
Another (better?) option might be to just use `uname -m` (aka `syscall.Uname` in golang).